sqldelight-gradle-plugin/src/test/fixtures/generate-view-mappers/expected/com/sample/Test2Model.java

```diff
@@ -159,17 +159,17 @@ public T map(@NonNull Cursor cursor) {
 
     private final Factory<V1T1> test2ModelFactory;
 
-    private final Test2_copyCreator<V1T1, V1> test2_copyCreator;
-
     private final Test1Model.Factory<V2T1> test1ModelFactory;
 
+    private final Test2_copyCreator<V1T1, V1> test2_copyCreator;
+
     private final Multiple_tablesCreator<V2T1, V1T1, V2> multiple_tablesCreator;
 
-    public Multiple_view_selectMapper(Multiple_view_selectCreator<V1T1, V1, V2T1, V2, T> creator, Factory<V1T1> test2ModelFactory, Test2_copyCreator<V1T1, V1> test2_copyCreator, Test1Model.Factory<V2T1> test1ModelFactory, Multiple_tablesCreator<V2T1, V1T1, V2> multiple_tablesCreator) {
+    public Multiple_view_selectMapper(Multiple_view_selectCreator<V1T1, V1, V2T1, V2, T> creator, Factory<V1T1> test2ModelFactory, Test1Model.Factory<V2T1> test1ModelFactory, Test2_copyCreator<V1T1, V1> test2_copyCreator, Multiple_tablesCreator<V2T1, V1T1, V2> multiple_tablesCreator) {
```

Comment:
this switches around factory order in more complicated queries so potential breaking change. Pretty small though